content/docs/architecture/auth.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Authentication and Authorization
+weight: 60
+---
+
+
+Fluvio uses mTLS (mutual TLS) for authentication between client and the fluvio cluster.   The user id (or principle) is extracted from the client certificate then apply to authorization policy.  Currently, Fluvio supports a simple RBAC (role based access control) authorization model but it could be extended to support more complex authorization scheme in the future. 
+
+# Cluster setup TLS
+
+First Fluvio must be running with TLS enabled. Once TLS is enabled, Both SC and SPU will accept mTLS base traffic from the client.  As part of TLS based configuration, admin can also specify authorization policy file.
+
+To create a Fluvio cluster with TLS enabled, you must provide following certs and keys:
+- CA cert
+- Client cert and key
+- Server cert and key
+- Authorization policy file
+
+For development and testing, you can use self-signed certs and keys.  For production, you should use certs and keys signed by a trusted CA.  It is important to note that the client cert and key must be signed by the same CA as the server cert and key.  Otherwise, the client will not be able to connect to the server.  In addition, admin must specify domain (DNS name) that cluster will be bind to.  This is required for TLS to work properly.
```

Comment:
<s> In addition, admin must specify domain (DNS name) that cluster will be bind to. </s>
In addition, the system admin must specify domain name (DNS) that cluster is bound to.

Reply:
updated